tests/test_connection.py

```diff
@@ -102,6 +102,34 @@ def test_new_connection_with_jwt_auth(runner, os_agnostic_snapshot):
     assert content == os_agnostic_snapshot
 
 
+def test_if_whitespaces_are_stripped_from_connection_name(runner, os_agnostic_snapshot):
+    with NamedTemporaryFile("w+", suffix=".toml") as tmp_file:
+        result = runner.invoke_with_config_file(
+            tmp_file.name,
+            [
+                "connection",
+                "add",
+            ],
+            input="       whitespaceTest     \naccName     \n     userName",
```

Comment:
Could you test all values?

Reply:
Done, now it tests both prompt and command line inputs